code/modules/assembly/igniter.dm

```diff
@@ -17,17 +17,18 @@
 	sparks = null
 	return ..()
 
-
 /obj/item/device/assembly/igniter/activate()
 	if(!..())
 		return 0//Cooldown check
 	var/turf/location = get_turf(loc)
 	if(location)
 		location.hotspot_expose(1000,1000)
+	if(istype(src.loc.loc,/obj/structure/reagent_dispensers/fueltank))
+		for (var/obj/structure/reagent_dispensers/fueltank/F in location)
+			F.boom()

```

Comment:
By making hotspot_expose trigger the fueltank, It would be hilarious to see what else would trigger fueltanks.